web/modules/custom/mymodule/src/Controller/FirstController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file 
+ * Generate markup to be displayed. Functionality in this controller is
+ * wired to Drupal in mymodule.routing.yml.
+ */
+
+namespace Drupal\mymodule\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+
+class FirstController extends ControllerBase {
+
+  public function simpleContent() {
```

Comment:
Please provide docblock for all the functions defined here.

Reply:
Provided.